server/swap/swap.go

```diff
@@ -954,16 +939,14 @@ func (s *Swapper) processRedeem(msg *msgjson.Message, params *msgjson.Redeem, st
 	counterParty.status.mtx.RLock()
 	cpSwapCoin := counterParty.status.swap.ID()
 	counterParty.status.mtx.RUnlock()
-	spends, err := coin.SpendsCoin(cpSwapCoin)
+
+	// Get the transaction
+	chain := stepInfo.asset.Backend
+	redemption, err := chain.Redemption(params.CoinID, cpSwapCoin)
+	// If there is an error, don't return an error yet, since it could be due to
+	// network latency. Instead, queue it up for another check.
```

Comment:
Seems this could have the same issue (network latency vs. does not spend the swap contract coin).